libs/auth-api-lib/seeders/data/scope-signature-collection.ts

```diff
@@ -29,7 +29,7 @@ export const up = compose(
   }),
   createScope({
     name: '@island.is/signature-collection',
-    displayName: 'Meðmæla söfnun',
+    displayName: 'Meðmælasöfnun',
```

Comment:
FYI this seed won´t be run again as the scope already exists. So please head over to #ids_changes channel on Slack and request the change for the display name.